src/fenicsxconcrete/experiment.py

```diff
@@ -0,0 +1,35 @@
+from pydantic import RootModel
+from bcs import DirichletBCDefinition, InitialConditionDefinition, NeumannBCDefinition, BodyForceDefinition
+from mesh import MeshGenerator
+from material import MaterialDefinition, LinearElasticMaterial
+from pydantic.dataclasses import dataclass
+
+@dataclass(config=dict(arbitrary_types_allowed=True))
+class Experiment:
+    dirichlet_bcs: list[DirichletBCDefinition] | None
+    neumann_bcs: list[NeumannBCDefinition] | None
+    initial_conditions: list[InitialConditionDefinition] | None
+    body_forces: list[BodyForceDefinition] | None
+    geometry: MeshGenerator
+    solution_fields: list[str]
+    time: tuple[float, float]
+    material: MaterialDefinition
+    name: str = "experiment"
+
+
+
+
+if __name__ == "__main__":
+    bc = DirichletBCDefinition(marker=1, value=[0.,0.], subspace=0, variable='displacement')
+    neumann = NeumannBCDefinition(marker=2, value=[-42.0], variable='displacement')
+    initial = InitialConditionDefinition(value=[42.24], variable='density')
+    body_force = BodyForceDefinition(value=[0.,0., 9.81], variable='displacement')
```

Comment:
How can we ensure that the body force is applied in the correct direction? We had some problems on that in the current version.

Reply:
Based on the users coordinate system and the three components of the body force I would expect that to be unique, what was the problem in the current version?